### ISSUE TITLE
Importing message structures from several HL7 versions is not possible

### ISSUE THREAD

**Issue by carolinerosin** [haffonist/igamt#1961]
In the eDOS guide, there are 3 message prodfiles based on HL7v2.5.1 and one pre-adopted from HL7 v2.8.2.
If I choose the HL7 version 2.5.1 when I create my IG, I can't add the message structure that is pre-adopted (it did not exist in v2.5.1).

Is that something that needs to be allowed in IGAMT ?

**Comment by robnist** [comment]
Yes we have to support that. The version id for the IG really is not that important. It is more important for the version ID of each profile. 

IHE often will mix versions in their integration profiles and I think we can expect this in may IGs.

I guess a similar feature that we have for pre-adopting (which is a bad term) that we use for segments need to be added for messages.

**Comment by afrold** [comment]
@Abdelghani90 
Please work on this  and hold off the work on the upgrade. 

Thanks
